example-studio-all-fashion-mnist/code/params.yaml

```diff
@@ -0,0 +1,7 @@
+train:
+  batch_size: 128
```

Comment:
We used to have many parameters but now we have two, the number of CNN units and the epochs. If you would like to have other parameters, I can add.